dependencies.yaml

```diff
@@ -170,10 +170,13 @@ dependencies:
     common:
       - output_types: [conda, requirements]
         packages:
+          - opencv
```

Comment:
Are all these new dependencies just for this notebook? Seems like a lot to impose on all users of cuSpatial if it's only for one example notebook.

Reply:
opencv is only used to visualize the cluster results. We should be able to mimic the behavior via shapely and matplotlib. Or as @jarmak-nv suggests geopandas can do that job.

Reply:
There are a few other added dependencies other than opencv below this.

Reply:
All dependencies here are only installed when testing notebooks. They are not dependencies for python/c++ builds.

I replaced opencv with matplotlib plotting but kept the rest. Ipywidget is lightweight and is helpful for manual hyperparameter tuning. Scikit-learn is the lightest weight image I/O package that I can find.